### ISSUE TITLE
swar is MIA for 2+weeks.

### ISSUE THREAD

**Issue by creamsoup** [swar/Swar-Chia-Plot-Manager#1153]
basically this project is dead if this continues. should fix this single point of failure by adding some maintainers or be more responsive.

**Comment by joaquimguimaraes** [comment]
What’s wrong with the current plot manager?
Nothing’s stopping you from improving it.

**Comment by creamsoup** [comment]
@joaquimguimaraes i am currently writing my own. 
yes, you can fork it and merge your changes on that forked repo. can't contributes back to the main if no maintainers.

**Comment by joaquimguimaraes** [comment]
You can submit a pull request. Swar checks those.

**Comment by creamsoup** [comment]
@joaquimguimaraes i already submitted a PR 15 days ago. for 19 days, no activities from swar.

**Comment by swar** [comment]
@creamsoup Yes no activities from me because my mailbox has been overloaded from emails so even any honest questions are getting lost in the shuffle.

Not only that but I have other things going on in life that will take precedence over supporting a free open-source project. It's completely usable where it is right now, I don't know why anyone is expected constant updates. Summer semester just started and that's going to be occupying my time for the unforeseeable future. Updates will happen, just don't expect them to happen immediately.

**Comment by creamsoup** [comment]
@swar i or the community didn't ask immediate response, 20+ pending PRs and lots of issues filed on the repo sounds like it is far from stable. yes, it is usable for someone with developing background.
i know you said you will update sometime in the future, but what i am really hearing is you are less willing to maintain the repository due to your personal situation. you should mention this in something like `CONTRIBUTING.md` to let the future contributors aware. in this way, some of them won't waste their time.

**Comment by sothix** [comment]
@swar Could you assign someone to help you merge the pull requests since your time is limited now?

**Comment by creamsoup** [comment]
@sothix that was my main point which i stated in the issue. apparently, it wasn't able to draw any attention.

**Comment by endurance1968** [comment]
Completely understand that someone gets overloaded. 
There is farmr.net and this is also using this piece of software. This manager for me works perfectly as standalone - before I put more efforts in it it should be known if it will be actively maintained - if not by @swar then by someone from the team being part of farmr.net.

**Comment by wolfrage76** [comment]
![gene hoffman](https://user-images.githubusercontent.com/75458290/122692967-c6fbeb00-d205-11eb-97be-614289be4462.jpg)
